src/status_im/contexts/chat/messenger/messages/scroll_to_bottom/style.cljs

```diff
@@ -1,9 +1,13 @@
 (ns status-im.contexts.chat.messenger.messages.scroll-to-bottom.style
-  (:require [status-im.contexts.shell.jump-to.constants :as shell.constants]))
+  (:require [react-native.safe-area :as safe-area]
+            [status-im.contexts.shell.jump-to.constants :as shell.constants]))
 
-(def shell-button-container
+(defn shell-button-container
+  [able-to-send-messages?]
   {:z-index 1
-   :bottom  shell.constants/floating-shell-button-height})
+   :bottom  (+ (safe-area/get-bottom)
+               (when-not able-to-send-messages? 46)
+               shell.constants/floating-shell-button-height)})
```

Comment:
This is the second fix about the scroll to bottom button in chats

Reply:
This 46 is suspicious @ulisesmac, do you know if it can be calculated to be the sum of other known values?

Reply:
Thanks for noticing @ilmotta 

This is the height of the bottom drawer that says "requst to join". It's specified in figma. I can name it, do you think that'd be better?

Reply:
It's a large and specific number @ulisesmac, that's why it caught my attention. Usually if it's 4, 8, 12, 20 we know they are just standard spacing values, but 46 I thought could be a magic number. Numbers that are tied to a specific component width/height are usually good candidates be defined as vars. One reason is that if designers change in Figma we won't be able to easily/safely adjust in the code.